script/hassfest/mypy_config.py

```diff
@@ -158,7 +189,7 @@ def generate_and_validate(config: Config) -> str:
     mypy_config.add_section(general_section)
     for key, value in GENERAL_SETTINGS.items():
         mypy_config.set(general_section, key, value)
-    for key in STRICT_SETTINGS:
+    for key in sorted(STRICT_SETTINGS):
```

Comment:
```suggestion
    for key in STRICT_SETTINGS:
```

Reply:
There is nothing to keep `STRICT_SETTINGS` sorted otherwise.

Reply:
Okiedoke. Removed that.